documentation/source/user_section/installation/linux.rst

```diff
@@ -154,52 +161,30 @@ Docker Image: Ubuntu
 .. code:: bash
 
    # Start from the Terminal
-   docker pull ubuntu:16.04
+   docker pull ubuntu:22.04
    # Launch interactive mode (command-line inside container)
-   docker run -it ubuntu
+   docker run -it ubuntu:22.04
    # Now, inside Docker container, install dependencies
    apt-get update
-   apt install -y git curl bzip2 libglib2.0-0 gcc
-   # Note for above: libglib2.0-0 is required by PyQt
-   # Install SCT
-   git clone https://github.com/spinalcordtoolbox/spinalcordtoolbox.git sct
-   cd sct
-   ./install_sct -y
-   export PATH="/sct/bin:${PATH}"
-   # Test SCT
-   sct_testing
-   # save the state of the container. Open a new Terminal and run:
-   docker ps -a  # list all containers
-   docker commit <CONTAINER_ID> <YOUR_NAME>/ubuntu:ubuntu16.04
-
-Docker Image: CentOS7
-^^^^^^^^^^^^^^^^^^^^^
-
-.. code:: bash
-
-   # Start from the Terminal
-   docker pull centos:centos7
-   # Launch interactive mode (command-line inside container)
-   docker run -it centos:centos7
-   # Now, inside Docker container, install dependencies
-   yum install -y which gcc git curl
+   apt install -y git curl bzip2 libglib2.0-0 libgl1-mesa-glx libxrender1 libxkbcommon-x11-0 libdbus-1-3 gcc
+   # Note for above: libglib2.0-0, libgl1-mesa-glx, libxrender1, libxkbcommon-x11-0, libdbus-1-3 are required by PyQt
    # Install SCT
    git clone https://github.com/spinalcordtoolbox/spinalcordtoolbox.git sct
    cd sct
    ./install_sct -y
-   export PATH="/sct/bin:${PATH}"
+   source /root/.bashrc
    # Test SCT
    sct_testing
    # save the state of the container. Open a new Terminal and run:
    docker ps -a  # list all containers
-   docker commit <CONTAINER_ID> <YOUR_NAME>/centos:centos7
+   docker commit <CONTAINER_ID> <YOUR_NAME>/ubuntu:ubuntu22.04
```

Comment:
This is kind of a nitpick, but `<YOUR_NAME>` might be a bit unclear? (If a user takes it extremely literally, they might put e.g. `joshua` instead of a useful image name :rofl:)

So, it might be better to explicitly state that the user is choosing the name for a docker image here:

```bash
   # save the state of the container as a docker image. 
   # Open a new terminal and run:
   docker ps -a  # list all containers (to find out the container ID)
   # specify the ID, and also choose a name to use for the docker image, such as "sct_v6.0"
   docker commit <CONTAINER_ID> <IMAGE_NAME>/ubuntu:ubuntu22.04
```

(I also updated the comments in this section to be a bit more explicit, too.)

((This would also require changing `<YOUR_NAME>` to `<IMAGE_NAME>` elsewhere in the documentation.))